### PR TITLE
adds ZoneShootAuto

### PR DESCRIPTION
**Pull request by zzberries**
This command is used during the autonomous shooting challenge to shoot
from different zone. The current values need to be updated after data is
collected.

### CODE REVIEW

File: src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            }
+            if (zone == "red") {
+                addCommands(
+                new PIDBottomShooter(0.6));
+            }
+    }
+
+    public static void registerWithTestingDashboard() {
+        Auto auto = Auto.getInstance();
+        ZoneShootAuto cmd = new ZoneShootAuto("hello");
```
Comment (snowpuppy) [inline src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java]: Please change "hello" to one of the zone colors. Also, please create and register this command multiple times. (Once for each zone) Doing multiple registrations will allow us to select the correct zone from testing dashboard.

Comment (snowpuppy) [reply src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java]: Specificially, I'm looking for something like this:
        Auto auto = Auto.getInstance();
        ZoneShootAuto greenZone = new ZoneShootAuto("green");
        ZoneShootAuto yellowZone = new ZoneShootAuto("yellow");
        ZoneShootAuto blueZone = new ZoneShootAuto("blue");
        ZoneShootAuto redZone = new ZoneShootAuto("red");
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", greenZone);
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", yellowZone);
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", blueZone);
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", redZone);

Comment (zzberries) [reply src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java]: In the RobotContainer, should it still stay the same as the original? If so, then I made all of the changes necessary.

Comment (snowpuppy) [reply src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java]: Yes, the RobotContainer should stay the same.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 import edu.wpi.first.wpilibj.trajectory.constraint.DifferentialDriveVoltageConstraint;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.RamseteCommand;
-
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
```
Comment (snowpuppy) [inline src/main/java/frc/robot/RobotContainer.java]: Was this import needed to compile? If it's unused, please remove it.

File: src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            }
+    }
+
+    public static void registerWithTestingDashboard() {
+        Auto auto = Auto.getInstance();
+        ZoneShootAuto greenZone = new ZoneShootAuto("green");
+        ZoneShootAuto yellowZone = new ZoneShootAuto("yellow");
+        ZoneShootAuto blueZone = new ZoneShootAuto("blue");
+        ZoneShootAuto redZone = new ZoneShootAuto("red");
+        TestingDashboard.getInstance().registerCommand(auto, "Auto", greenZone);
```
Comment (snowpuppy) [inline src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java]: We don't need to register each command twice to testing dashboard. Please remove the 4 lines registering with the "Auto" group name and leave the ones registering with the "ZoneShootAuto" group name.

**Review by snowpuppy — CHANGES_REQUESTED** [review]

**Pull request merged**